services/credential-server/src/utils/lmdb.ts

```diff
@@ -0,0 +1,39 @@
+import { open, RootDatabase } from "lmdb";
+
+class Lmdb {
```

Comment:
Usually it's better to not name these kinds of classes based on the underlying tech. So `Database` would be more appropriate so that you could change from lmdb to something else without many code changes across the repo.